src/document/UBDocumentController.cpp

```diff
@@ -2316,6 +2316,7 @@ void UBDocumentController::setupViews()
 
         connect(mDocumentUI->sortKind, SIGNAL(activated(int)), this, SLOT(onSortKindChanged(int)));
         connect(mDocumentUI->sortOrder, SIGNAL(toggled(bool)), this, SLOT(onSortOrderChanged(bool)));
+        connect(mDocumentUI->filterText, SIGNAL(textChanged(const QString&)), this, SLOT(onFilterTextChanged(const QString&)));
```

Comment:
I prefer the new syntax for making connection using function pointers and would like to use it in any new code. This already provides compile time checks for signals and slots. It also makes it easier to find references to these signals and slots in QtCreator.

Reply:
Done